bower.json

```diff
@@ -1,18 +0,0 @@
-{
```

Comment:
They can depend on the old version. Bower is officially deprecated: https://github.com/bower/bower/issues/2298